address/multisig_address.go

```diff
@@ -9,20 +9,45 @@ import (
 	"github.com/nervosnetwork/ckb-sdk-go/types"
 )
 
-func GenerateSecp256k1MultisigScript(requireN, threshold int, publicKeys [][]byte) (*types.Script, []byte, error) {
+// GenerateSecp256k1MultisigScript generate scep256k1 multisig script.
+// It can accept public key (in compressed format, 33 bytes each) array or public key hash (20 bytes) array, and
+// return error if giving none of them.
+func GenerateSecp256k1MultisigScript(requireN, threshold int, publicKeysOrHashes [][]byte) (*types.Script, []byte, error) {
 	if requireN < 0 || requireN > 255 {
 		return nil, nil, errors.New("requireN must ranging from 0 to 255")
 	}
 	if threshold < 0 || threshold > 255 {
 		return nil, nil, errors.New("requireN must ranging from 0 to 255")
 	}
-	if len(publicKeys) > 255 {
+	if len(publicKeysOrHashes) > 255 {
 		return nil, nil, errors.New("public keys size must be less than 256")
 	}
-	if len(publicKeys) < requireN || len(publicKeys) < threshold {
+	if len(publicKeysOrHashes) < requireN || len(publicKeysOrHashes) < threshold {
 		return nil, nil, errors.New("public keys error")
 	}
 
+	isPublicKeyHash := len(publicKeysOrHashes[0]) == 20
```

Comment:
please move this block to to the line 67, it's unnecessary to create a temp array `publicKeysHash`.